regtests/ww3_tp2.14/input/oasis3-mct/util/make_dir/cmplr.tmpl

```diff
@@ -30,8 +30,8 @@ COUPLE          = $(WWATCH3_DIR)/../regtests/ww3_tp2.14/input/oasis3-mct
 ARCHDIR         = $(WWATCH3_DIR)/../regtests/ww3_tp2.14/work_oasis3-mct
 #
 # NetCDF library
-NETCDF_INCLUDE  = $(shell $(NETCDF_CONFIG) --includedir) 
-NETCDF_LIBRARY  = $(shell $(NETCDF_CONFIG) --flibs) 
+NETCDF_INCLUDE  = $(shell $(NETCDF_CONFIG) --cflags) 
```

Comment:
--cflags is for c-compiler
for fortran-compiler, it must be --fflags to be used